cernopendata/config.py

```diff
@@ -355,11 +355,12 @@ def _query_parser_and(qstr=None):
         )
     else:
         _query = dsl.Q()
-    if (
-        request.values.get("ondemand") != "true"
-        and request.values.get("ondemand") != "ondemand"
-    ):
-        _query = _query & ~dsl.Q("match", **{"distribution.availability": "ondemand"})
+
+    #    if (
```

Comment:
can be deleted?

Reply:
I've put it back, since the tests were not happy. If we remove this, we break backward compatibility if a user was searching through this field